File: BasicWidget/app/src/main/java/com/pyo/android/simple/widget/AndroidSimpleWidget.java
package com.pyo.android.simple.widget;


public class AndroidSimpleWidget extends BaseActivity {

    @Override
	public void displayActivityList() {
		addActionMap("1.TextView ", TextViewDisplay.class);
		addActionMap("2.Form UI ",WidgetFormDisplay.class);
		addActionMap("3.Event 처리 ", AndroidEventListActivity.class);
	}
}

File: fridaypractice/android0319/BasicWidget/app/src/main/java/kr/co/mapo/basicwidget/AndroidSimpleWidget.kt
package kr.co.mapo.basicwidget

import TextViewDisplay
import com.pyo.android.simple.widget.AndroidEventListActivity

abstract class AndroidSimpleWidget : BaseActivity() {
    override fun displayActivityList(){
        addActionMap("1.TextView", TextViewDisplay::class.java)
        addActionMap("2.Form UI",WidgetFormDisplay::class.java)
        addActionMap("3.Event 처리",AndroidEventListActivity::class.java)
    }
}